tests/platforms/windows/app/test_package.py

```diff
@@ -207,6 +226,19 @@ def test_package_msi(package_command, first_app_config, tmp_path):
     ]
 
 
+def test_package_zip(package_zip_command, first_app_config, tmp_path):
+    """A Windows app can be packaged as a zip file."""
+
+    first_app_config.packaging_format = "zip"
+    package_zip_command.package_app(first_app_config)
+
+    # The zip file exists and contains the content of the src folder
+    archive_file = tmp_path / "base_path" / "dist" / "First App-0.0.1.zip"
+    assert archive_file.exists()
+    with ZipFile(archive_file) as archive:
+        assert archive.namelist() == ["First App.exe"]
```

Comment:
It does not only give you coverage, it will also show you that creating a zip folder with the correct name does work.
My first guess was that within the testing environment I find some real hello world project that is used for building, running, packaging and which I can use. When I realized that e.g. the msi packager tests does not actually create an MSI installer, I wondered how important testing the zip content is considered crucial here. Of course, I can put the correct folder structure in the zip and add some more mock files, still, this wouldn't show that the packed app will run.